### PR TITLE
Texture Settings UI refactor, fixes, and improvements.

### PR DESCRIPTION
**Pull request by micronAMZN**
- Layout is now side-by-side.
- Texture preview is 512x512.
- Broken resources are replaced with UI 2.0 SVGs.
- All the features are now available including mipmap visualization, auto and manual refresh, info display, and reset to defaults.
- Known issues and many unreported issues fixed.
- UI string improvements and tool tip text improvements and additions.
- Help button now links to forthcoming Texture Settings help topic instead of top level o3de.org/docs.

The image below is update based on the [latest commit](https://github.com/o3de/o3de/pull/7729/commits/a48f6b927b7e6c59e569cf0b749cb0927a0029be)

LatestTextureSettings

New updates based on UI/UX feedback:

* Interface slightly smaller.
* Aligned non-reflected properties and labels.
* Moved Help button.
* Removed file masks from interface. They are in the preset info now.
* Moved Pixel Sampler and Filter Type properties to the top of mipmap settings so they aren't assumed to only affect alpha channels.
* Made all check boxes toggle switches.
* Removed the file masks filter check. It wasn't working and it makes more sense for every preset to have unique file masks so the appropriate preset is selected automatically.
* Moved the settings up to provide more room for mipmap settings.
* Preset info window automatically sizes to content.
* Further improvements to UI strings and tooltips.

Signed-off-by: Mike Cronin <58789750+micronAMZN@users.noreply.github.com>

### CODE REVIEW

File: Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Editor/TexturePreviewWidget.cpp [hunk 1]
```diff
@@ -95,21 +98,23 @@ namespace ImageProcessingAtomEditor
         menu->addAction(m_refreshPerClickAction);
 
         m_ui->refreshBtn->setMenu(menu);
-        AzQtComponents::PushButton::applySmallIconStyle(m_ui->refreshBtn);
+        //AzQtComponents::PushButton::applySmallIconStyle(m_ui->refreshBtn);
```
Comment (AMZN-daimini) [inline Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Editor/TexturePreviewWidget.cpp:101]: nit: remove commented code

**Review by AMZN-daimini — APPROVED** [review]

**Comment by rainbj** [comment]
After reviewing the layout with the Sig-UX-Ui group this morning we noticed that all the fields were not on a grid and it looks a bit messy.  Take a look at the below image and let me know if you have any questions. @micronAMZN
layout updates

**Comment by rainbj** [comment]
@micronAMZN can you set up some time with @o3de/sig-ui-ux to talk about this item. Thanks

**Comment by micronAMZN** [comment]
Due to a stupid merge/force reset mistake, I just lost the last several days of work I put into addressing this feedback. I can't invest anything more into this PR to redo this work at this time and have to move on.

A few notes.

* I couldn't set the save button to auto default (highlighted). `setAutoDefault(true)` had no effect.
* The title bar help style doesn't seem to be implemented at the framework level. I couldn't get it to work, nor could I find a working example in O3DE. I put the help button in what seemed to be the next most logical place.
* This interface is spread over about 30 files and parts of it are automatically generated with `ReflectedPropertyEditor`. Because there are many different classes and layouts used, even doing simple things like moving the **Use Max Res** toggle can take considerable effort because all of its functionality has to be moved between classes.
* Some parts of the interface require a lot of space so that they can accommodate very long strings found in the preset files.
* Trying to line up all the elements on a gig-line requires a lot of trial and error due to the varying layouts used between property groups. There are still elements in the UI files that aren't necessary and add further complications.  
* The arrow cycles through the mipmap levels. Another arrow appears on the left when the user steps down the chain, allowing them to step back up the mip chain. It's a pretty important feature and probably the main reason I started making changes to the interface.

**Comment by AMZN-daimini** [comment]
> * I couldn't set the save button to auto default (highlighted). `setAutoDefault(true)` had no effect.

If you have some info, you can report this as a bug so it gets looked at by editor.

> * The title bar help style doesn't seem to be implemented at the framework level. I couldn't get it to work, nor could I find a working example in O3DE. I put the help button in what seemed to be the next most logical place.

Yeah this is not a style we support on the window titlebar right now. We can add it relatively easily I'd wager. @rainbj can you add a feature request on the editor side?

> * This interface is spread over about 30 files and parts of it are automatically generated with `ReflectedPropertyEditor`. Because there are many different classes and layouts used, even doing simple things like moving the **Use Max Res** toggle can take considerable effort because all of its functionality has to be moved between classes.

By the looks of it, it's mostly a matter of the second block of options being nested? So yeah that's how the RPE behaves in that situation. you can probably hide the parent to have everything show aligned as requested, but then you'd lose an option unless something is refactored.

**Comment by micronAMZN** [comment]
I'll try to spend a bit more time on this over the next two days.

**Comment by micronAMZN** [comment]
Here is the latest version. I have tried to make uniform margins, and integrated as much of the feedback as I could.

newestlatesttexturesettings

**Review by cgalvan — APPROVED** [review]

**Pull request merged**